.github/workflows/ci.yml

```diff
@@ -460,50 +391,46 @@ jobs:
                                       --binary="$DENO_BIN"          \
                                       --json=wpt.json               \
                                       --wptreport=wptreport.json
-
       - name: Upload wpt results to dl.deno.land
         continue-on-error: true
-        if: |
+        if: |-
           runner.os == 'Linux' &&
           matrix.job == 'test' &&
           matrix.profile == 'release' &&
           github.repository == 'denoland/deno' &&
           github.ref == 'refs/heads/main' && !startsWith(github.ref, 'refs/tags/')
-        run: |
+        run: |-
           gzip ./wptreport.json
           gsutil -h "Cache-Control: public, max-age=3600" cp ./wpt.json gs://dl.deno.land/wpt/$(git rev-parse HEAD).json
           gsutil -h "Cache-Control: public, max-age=3600" cp ./wptreport.json.gz gs://dl.deno.land/wpt/$(git rev-parse HEAD)-wptreport.json.gz
           echo $(git rev-parse HEAD) > wpt-latest.txt
           gsutil -h "Cache-Control: no-cache" cp wpt-latest.txt gs://dl.deno.land/wpt-latest.txt
-
       - name: Upload wpt results to wpt.fyi
         continue-on-error: true
-        if: |
+        if: |-
           runner.os == 'Linux' &&
           matrix.job == 'test' &&
           matrix.profile == 'release' &&
           github.repository == 'denoland/deno' &&
           github.ref == 'refs/heads/main' && !startsWith(github.ref, 'refs/tags/')
         env:
           WPT_FYI_USER: deno
-          WPT_FYI_PW: ${{ secrets.WPT_FYI_PW }}
-          GITHUB_TOKEN: ${{ secrets.DENOBOT_PAT }}
-        run: |
+          WPT_FYI_PW: '${{ secrets.WPT_FYI_PW }}'
+          GITHUB_TOKEN: '${{ secrets.DENOBOT_PAT }}'
+        run: |-
           ./target/release/deno run --allow-all --lock=tools/deno.lock.json \
               ./tools/upload_wptfyi.js $(git rev-parse HEAD) --ghstatus
-
       - name: Run benchmarks
-        if: matrix.job == 'bench' && !startsWith(github.ref, 'refs/tags/')
+        if: 'matrix.job == ''bench'' && !startsWith(github.ref, ''refs/tags/'')'
```

Comment:
This two single quotes looks strange there around `bench`

Reply:
Yeah, I'm not sure what I can do with deno_std to get this to be left alone

Reply:
The important thing is more the ci.generate.ts because we're not maintaining this file anymore.